### DISCUSSION TITLE
customized SO_CAL and Tail number/callsign in cockpit per each repaint

### DISCUSSION THREAD

**Discussion by AndyCYXU** [flybywiresim/aircraft#6945] [Ideas]
some aircraft "Aerosoft CRJ" can have separate tail/callsign number for each repaint in the cockpit.. can you make it like this too?

 I know I can edit the DECALS,png but that wont work if I put it in each aircraft repaint, well didn't seem to for me, only the one in main a320 texture directory is taken into account

I am taking about the callsign and the SO-CAL displayed in the cockpit

would be nice to have each repaint have its own custom tail number/callsign and so-cal code in the cockpit

**Comment by tracernz** [comment]
Most livery creators are already doing this. You need to put the decal texture and it’s .json file in the repaint package, and re-generate layout.json for the package.

**Reply by AndyCYXU** [reply]
OK, Thanks I will give that a try again, I noticed I didn't add the a320*decal*_json file form the texture itself and... might not have actually run the main json file through msfs_json generator.

**Comment by AndyCYXU** [comment]
Great, it works ! Thanks !
